test/transcoding/fp_contract_reassoc_fast_mode.ll

```diff
@@ -0,0 +1,50 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv -spirv-text %t.bc -o - | FileCheck %s --check-prefix=CHECK-SPIRV
```

Comment:
Test should be adjusted to incorporate the fact that the new extension might be disabled and must be enabled in order to take effect